src/features/Tiles/PeopleTiles/index.js

```diff
@@ -42,14 +44,15 @@ const PeopleTiles = ({ title, creditsData }) => {
                 <Title>{title}</Title>
                 <PeopleList>
                     {peopleList.map((person, index) =>
-                        <li key={index}>
+                        <li
+                            key={index}
+                            hidden={
+                                !sectionName.includes("popular") &&
+                                !showMore &&
+                                index > 5
+                            }>
                             <PersonTile
                                 to={`/profile/${person.id}`}
-                                hidden={
-                                    !sectionName.includes("popular") &&
-                                    !showMore &&
-                                    index > 5
-                                }
                             >
                                 <ProfilePicture
                                     src={person.profile_path ?
```

Comment:
Because We use person.profile_path in more than one place it would be good to name and declare as const and use we its necesary.

Reply:
Yes, I get your point but I don't really know how to do it. Notice person is in a map function so It's a variable name for every element of peopeList. 
I can do destructuring:
```
 {peopleList.map(({ id, profile_path, name, character, job }, index) =>
                        <li
                            key={index}
                            hidden={
                                !sectionName.includes("popular") &&
                                !showMore &&
                                index > 5
                            }>
                            <PersonTile
                                to={`/profile/${id}`}
                            >
                                <ProfilePicture
                                    src={profile_path ?
                                        `${pictureLinkBase}${profile_path}` :
                                        pictureSubstitution}
                                />
                                <TileTitle>{name}</TileTitle>
                                {
                                    !sectionName.includes("popular") &&
                                    <Role>
                                        {character || job}
                                    </Role>
                                }
                            </PersonTile>
                        </li>
                    )}
```
Do you like it more? Or is there a better way to do it? Let me know and I will change it :)

Reply:
great! the coding with destructurisation looks very good.